packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -194,6 +194,20 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
     throw UnimplementedError('animateCamera() has not been implemented.');
   }
 
+  /// Starts an animated change of the map camera position using the provided
+  /// [CameraUpdateAnimationConfiguration].
+  ///
+  /// The returned [Future] completes after the change has been started on the
+  /// platform side.
+  Future<void> animateCameraWithConfiguration(
+    CameraUpdate cameraUpdate, {
+    required CameraUpdateAnimationConfiguration configuration,
+    required int mapId,
+  }) {
+    throw UnimplementedError(
+        'animateCameraWithConfiguration() has not been implemented.');
```

Comment:
This should call `animateCamera`; see https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changing-platform-interface-method-parameters

Reply:
And the forwarding should get a unit test in this package.